daml/Marketplace/Registry/Issuer.daml

```diff
@@ -26,10 +26,8 @@ template Issuer
           newTitle : Text
           newIssuerID : Text
         do
-          create this with name = newName, location = newLocation, title = newTitle, issuerID = newIssuerID
+          create this with name = newName; location = newLocation; title = newTitle; issuerID = newIssuerID
```

Comment:
I prefer `,`. I find it really annoying that DAML has two ways of doing this!

Reply:
I used to prefer `,` but I wanted to be consistent with what @GeorgSchneider had done in the project and now I prefer `;` as I'm used to it now ...